mods/cnc/chrome/missionbrowser.yaml

```diff
@@ -68,6 +68,44 @@ Container@MISSIONBROWSER_PANEL:
 									IgnoreMouseOver: True
 									IgnoreMouseInput: True
 									ShowSpawnPoints: False
+						Container@MISSION_MINI_OPTIONS:
```

Comment:
`MISSION_BUTTONS` to go with `MISSION_TABS` below?

Reply:
changed to `Container@MISSION_MINIFIED_OPTIONS:`